### DISCUSSION TITLE
Reasons why I can't use your custom voice labels

### DISCUSSION THREAD

**Discussion by Falcosc** [HadrienGardeur/web-speech-recommended-voices#30] [General]
Thank you so much for all the work. I did not understand where your motivation is coming from. Just being the first for solving "European Accessibility Act in June 2025" would not be a motivation for me to put as much time into this like you did.

To boost your motivation level in case you need a boost: You get big Thank You from me!

Reasons for skipping your labels:
- Online tag missing
- Multilingual tag missing

Unfortunately, I can't use your custom voice labels on Microsoft Edge, since it does not contain the information "multilingual" and "online".
For all other browser, your labels are great and not a downgrade. Currently, I suggest Edge because it has most voices and most of them are high quality (maybe even the best? Did not test others). And because I focus on Edge, I did not even implement your voice labels at all, since I would only display them if not using edge.
And without these attributes, I can't extend your great labels with Online and Multilingual tags in my Picklist.

**Comment by HadrienGardeur** [comment]
Hello @Falcosc,

Thank you for your good words, it's always nice to hear that this is helpful! Do you mind sharing where/how you plan on using the voices documented in this project?

For Microsoft Edge, I've tested everything that I've documented on macOS but haven't found the time yet to do the same on Windows. Are you testing things on Windows on your side?

Based on my experience working with Edge and Windows voices, they tend to follow a pattern. If there's a variation in the pattern used by Edge on Windows, I'll gladly documented that in this project.

**Reply by Falcosc** [reply]
Yes, your names match. But we don't have enough attributes and the label does not tell what's behind that voice.

```
      "label": "Seraphina (Deutschland)",
      "name": "Microsoft SeraphinaMultilingual Online (Natural) - German (Germany)",
```

"Seraphina (Deutschland)" is not enough, I want to expose attributes like Multilingual support and Online Service

Since Edge has the most usable voices (give people in chat different voices) I focus my recommendations to edge.
But without the tags, I skip your label and use "Microsoft SeraphinaMultilingual Online (Natural) - German (Germany)"

For all other browsers, your labels are better than names.

**Reply by HadrienGardeur** [reply]
OK I see what you mean.

Multilingual support is identified using a property (`multiLingual`) and online/offline can be obtained through `localService ` when you use `getVoices()`. You could easily concatenate an additional string to the voice names that I provide or use icons if that works best for you.

I wouldn't recommend using the default `name`/`uri` because language and country names in English are not acceptable defaults IMO.

**Reply by Falcosc** [reply]
I did not see the multilingual property yesterday. Thanks for the tip with localService.

**Reply by Falcosc** [reply]
I found the reason for my confusion: multiLingual is an optional property. That's how I missed it.

**Reply by HadrienGardeur** [reply]
Yeah it's optional and fairly new (limited to a few voices available in Edge).

I've also identified that a few voices are capable of handling another language, even if they're not fully multilingual. I'll need to do more extensive testing to document `otherLanguages` on more voices.
